'ARM_INS_ERET' not in /usr/include/capstone/arm.h

```console
$ cat /etc/os-release 
NAME="Linux Mint"
VERSION="20.3 (Una)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 20.3"

$ sudo apt install libusb-1.0-0-dev libczmq-dev libncurses-dev libsdl2-dev libelf-dev libcapstone-dev

$ meson --version
1.4.0

$ git log
commit 7968f96ef92a00b42123bf794442d54005284db3 (HEAD -> main, origin/main, origin/HEAD)
Author: Vegard Storheil Eriksen <zyp@jvnv.net>
Date:   Tue Mar 26 00:23:45 2024 +0100

    Python interface moved to pyorb repo.
```

`meson setup build` is successful

<details>
  <summary>Click to expand</summary>
  
```console
$ meson setup build
The Meson build system
Version: 1.4.0
Source dir: /home/alex/git/orbuculum
Build dir: /home/alex/git/orbuculum/build
Build type: native build
Project name: orbuculum
Project version: 2.1.0
C compiler for the host machine: ccache cc (gcc 9.4.0 "cc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0")
C linker for the host machine: cc ld.bfd 2.34
Host machine cpu family: x86_64
Host machine cpu: x86_64
Run-time dependency threads found: YES
Found pkg-config: YES (/usr/bin/pkg-config) 0.29.1
Run-time dependency libusb-1.0 found: YES 1.0.23
Run-time dependency libzmq found: YES 4.3.2
Run-time dependency sdl2 found: YES 2.0.10
Run-time dependency ncurses found: YES 6.2.20200212
Run-time dependency capstone found: YES 3.0.5
Run-time dependency libelf found: YES 0.176

Executing subproject libdwarf 

libdwarf| Project name: libdwarf
libdwarf| Project version: 0.7.0
libdwarf| C compiler for the host machine: ccache cc (gcc 9.4.0 "cc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0")
libdwarf| C linker for the host machine: cc ld.bfd 2.34
libdwarf| C++ compiler for the host machine: ccache c++ (gcc 9.4.0 "c++ (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0")
libdwarf| C++ linker for the host machine: c++ ld.bfd 2.34
libdwarf| Compiler for C supports arguments -Wpointer-arith: YES
libdwarf| Compiler for C supports arguments -Wmissing-declarations: YES
libdwarf| Compiler for C supports arguments -Wstrict-prototypes: YES
libdwarf| Compiler for C supports arguments -Wcomment: YES
libdwarf| Compiler for C supports arguments -Wformat: YES
libdwarf| Compiler for C supports arguments -Wuninitialized: YES
libdwarf| Compiler for C supports arguments -Wshadow: YES
libdwarf| Compiler for C supports arguments -Wno-long-long: YES
libdwarf| Compiler for C supports arguments -Wfloat-compare: NO
libdwarf| Compiler for C supports arguments -Wsign-compare: YES
libdwarf| Compiler for C supports arguments -Wno-missing-field-initializers: YES
libdwarf| Compiler for C supports arguments -fno-omit-frame-pointer: YES
libdwarf| Compiler for C supports arguments -fvisibility=hidden: YES
libdwarf| Compiler for C supports function attribute unused: YES
libdwarf| Has header "fcntl.h" : YES
libdwarf| Has header "inttypes.h" : YES
libdwarf| Has header "malloc.h" : YES
libdwarf| Has header "stdint.h" : YES
libdwarf| Has header "sys/stat.h" : YES
libdwarf| Has header "unistd.h" : YES
libdwarf| Checking for type "uint64_t" : YES
libdwarf| Checking for type "uintptr_t" : YES
libdwarf| Checking for type "intptr_t" : YES
libdwarf| Has header "libelf.h" : YES
libdwarf| Has header "libelf/libelf.h" : NO
libdwarf| Has header "elf.h" : YES
libdwarf| Run-time dependency zlib found: YES 1.2.11
libdwarf| Run-time dependency libzstd found: NO (tried pkgconfig)
libdwarf| subprojects/libdwarf-0.7.0/test/meson.build:112: WARNING: Project targets '>=0.56' but uses feature deprecated since '0.56.0': meson.source_root. use meson.project_source_root() or meson.global_source_root() instead.
libdwarf| Program python3 found: YES (/home/alex/.local/pipx/venvs/meson/bin/python)
libdwarf| Message: Elf
libdwarf| subprojects/libdwarf-0.7.0/test/meson.build:137: WARNING: Project targets '>=0.56' but uses feature deprecated since '0.56.0': meson.build_root. use meson.project_build_root() or meson.global_build_root() instead.
libdwarf| Message: PE
libdwarf| Message: Macos
libdwarf| Program sh found: YES (/usr/bin/sh)
libdwarf| Configuring config.h using configuration
libdwarf| Configuring libdwarf.pc using configuration
libdwarf| Build targets in project: 26
libdwarf| WARNING: Deprecated features used:
libdwarf| * 0.56.0: {'meson.build_root', 'meson.source_root'}
libdwarf| Subproject libdwarf finished.

Build targets in project: 39

libdwarf 0.7.0

  Configuration Options Summary:
    OS               : linux
    BuildOS-BigEndian: no
    libelf support   : yes
    libdwarf         : always (zlib: yes) always (libzstd: no)
    dwarfdump        : always
    libdwarfp        : no
    dwarfgen         : no
    dwarfexample     : no
    documentation    : no

  Directories:
    prefix           : /usr/local
    bindir           : /usr/local/bin
    libdir           : /usr/local/lib/x86_64-linux-gnu
    incdir           : /usr/local/include
    pkgincdir        : /usr/local/include/libdwarf
    datadir          : /usr/local/share
    pkgdatadir       : /usr/local/share/libdwarf

  Compilation
    compilation      : ninja
    installation     : ninja install

orbuculum 2.1.0

  Subprojects
    libdwarf: YES 3 warnings

Found ninja-1.10.0 at /usr/bin/ninja
```

</details>

```console
$ ninja -C build
[....]

FAILED: orbmortem.p/Src_loadelf.c.o 
ccache cc -Iorbmortem.p -I. -I.. -I../Inc -I../Inc/external -Isubprojects/libdwarf-0.7.0/src/lib/libdwarf -I../subprojects/libdwarf-0.7.0/src/lib/libdwarf -I/usr/include/libusb-1.0 -I/usr/include/pgm-5.2 -I/usr/include/SDL2 -I/usr/include/capstone -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O0 -g -DLINUX -ggdb -D_GNU_SOURCE -DSCREEN_HANDLING -Wno-error=deprecated-declarations -include uicolours_default.h -include uicolours_default.h -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -D_REENTRANT -isystem /usr/include/mit-krb5 -pthread -MD -MQ orbmortem.p/Src_loadelf.c.o -MF orbmortem.p/Src_loadelf.c.o.d -o orbmortem.p/Src_loadelf.c.o -c ../Src/loadelf.c
../Src/loadelf.c: In function ‘symbolDisassembleLine’:
../Src/loadelf.c:1042:33: error: ‘ARM_INS_ERET’ undeclared (first use in this function); did you mean ‘ARM64_INS_ERET’?
 1042 |         *ic |=  ( ( insn->id == ARM_INS_ERET ) ) ? LE_IC_JUMP | LE_IC_IRET : 0;
      |                                 ^~~~~~~~~~~~
      |                                 ARM64_INS_ERET
../Src/loadelf.c:1042:33: note: each undeclared identifier is reported only once for each function it appears in
[261/271] Compiling C object orbtrace.p/Src_symbols.c.o
../Src/symbols.c:120:36: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
  120 |     #pragma GCC diagnostic ignored "-Wanalyzer-malloc-leak"
      |                                    ^~~~~~~~~~~~~~~~~~~~~~~~
../Src/symbols.c:1064:36: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
 1064 |     #pragma GCC diagnostic ignored "-Wanalyzer-malloc-leak"
      |                                    ^~~~~~~~~~~~~~~~~~~~~~~~
[263/271] Compiling C object orblcd.p/Src_orblcd.c.o
ninja: build stopped: subcommand failed.
```

Do I have an older version of something?

--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,6 @@ dependencies = [
     dependency('libzmq'),
     dependency('SDL2'),
     dependency('ncurses', 'ncursesw'),
-    dependency('capstone'),
     dependency('libelf'),
     uicolours_default,
     sockets,
@@ -46,6 +45,11 @@ add_project_arguments(['-include', 'uicolours_default.h'], language: 'c')
 
 libdwarf = subproject('libdwarf').get_variable('libdwarf')
 dependencies += libdwarf
+
+libcapstone = dependency('capstone', version: '>=4', required: false)
+if not libcapstone.found()
+    libcapstone = disabler()
+endif
 
 if host_machine.system() == 'windows'
     stream_src = [
@@ -173,7 +177,9 @@ executable('orbmortem',
         git_version_info_h,
     ],
     include_directories: incdirs,
-    dependencies: dependencies,
+    dependencies: dependencies + [
+        libcapstone,
+    ],
     link_with: liborb,
     install: true,
 )

